src/main/java/by/Starleken/repositories/RealizationCommentRepository.java

```diff
@@ -0,0 +1,98 @@
+package by.Starleken.repositories;
+
+import by.Starleken.entities.Comment;
+import by.Starleken.repositories.interfaces.CommentRepository;
+import by.Starleken.utils.EntityManagerUtils;
+import jakarta.persistence.EntityManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class RealizationCommentRepository implements CommentRepository {
```

Comment:
Please rename `Realization{EntityName}Repository` classes to `{EntityName}RepositoryImpl`

Impl means Implementation

`{EntityName}RepositoryImpl` is the default java naming convension, you gonna se the same in real projects